dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers/map-to-fee-record-utilisation.test.ts

```diff
@@ -51,31 +51,40 @@ describe('map-to-fee-record-utilisation', () => {
       tfmFacility.facilitySnapshot.coverPercentage = coverPercentage;
 
       // Act
-      const feeRecordUtilisation = mapToFeeRecordUtilisation(feeRecord, tfmFacility);
+      const feeRecordUtilisation = mapToFeeRecordUtilisation(feeRecord, tfmFacility, aReportPeriod());
 
       // Assert
       expect(feeRecordUtilisation.coverPercentage).toEqual(coverPercentage);
     });
 
     it('should check for amendments to the facility value', () => {
       // Arrange
-      const value = 200000;
-      const facilityId = '12345678';
+      const feeRecord = FeeRecordEntityMockBuilder.forReport(aUtilisationReport()).build();
+      const tfmFacility = aTfmFacility();
 
-      const feeRecord = FeeRecordEntityMockBuilder.forReport(aUtilisationReport()).withFacilityId(facilityId).build();
+      const getValueAmendmentSpy = jest.spyOn(helpers, 'getEffectiveFacilityValueAmendment').mockReturnValue(null);
 
-      const tfmFacility = aTfmFacility();
-      tfmFacility.facilitySnapshot.ukefFacilityId = facilityId;
-      tfmFacility.facilitySnapshot.value = value;
+      // Act
+      mapToFeeRecordUtilisation(feeRecord, tfmFacility, aReportPeriod());
+
+      // Assert
+      expect(getValueAmendmentSpy).toHaveBeenCalledTimes(1);
+      expect(getValueAmendmentSpy).toHaveBeenCalledWith(tfmFacility, expect.any(Date));
```

Comment:
Are we able to assert a real date value?

Reply:
We do that in the test below. My thought was it would be better to assert the different params called specifically in separate tests to split assertions and keep the tests more specific but I can combine into just one test if that would be preferred?

Reply:
Thanks @RLavender98 makes sense as it is